uptycs/manifest.json

```
@@ -0,0 +1,40 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d27ee4b6-649d-42bd-b7ac-fb40537d7031",
+  "app_id": "uptycs",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect alerts, detections data from Uptycs platform",
+    "title": "Uptycs",
+    "media": [
+     {
+        "media_type": "image",
+        "caption": "Uptycs logo",
+        "image_url": "logos/uptycs-logo.png"
+     }
+    ],
+    "classifier_tags": [
+      "Category::Cloud",
+      "Category::Collaboration",
+      "Category::Alerting",
+      "Category::Compliance"
```

```suggestion
      "Category::Compliance",
      "Category::Security",
      "Offering::Integration",
      "Submitted Data Type::Events"
```

A few suggestions. Feel free to adjust as needed, and add more classifier tags from the [list here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags).